sycl/doc/CompilerAndRuntimeDesign.md

```diff
@@ -787,18 +787,61 @@ The SPIR-V specific functions are implemented in for the SYCL host device here:
 
 ### Address spaces handling
 
-SYCL 1.2.1 language defines several address spaces where data can reside, the
-same as OpenCL - global, local, private and constant. From the spec: "In OpenCL
```

Comment:
Is it worth explicitly mentioning that we avoid using `constant` address space in our toolchain? I think it is kind of a design question/decision

Reply:
Done.